src/Features/CustomerEffortScoreTracks.php

```diff
@@ -129,6 +165,19 @@ public function run_on_transition_post_status(
 		}
 	}
 
+	/**
+	 * Maybe enqueue the CES survey, if shop order is being edited.
+	 *
+	 * @param string $new_status The new status.
+	 * @param string $old_status The old status.
+	 */
+	private function maybe_enqueue_ces_survey_for_shop_order(
```

Comment:
This function uses a `maybe_` prefix, but the logic and condition for this appear to be in `enqueue_ces_survey_for_edited_shop_order`.

Reply:
Seems like a remnant of an earlier version of the implementation. I will clean this up before merging.